src/Impostor.Api/Events/Announcements/IAnnouncementRequestEvent.cs

```diff
@@ -0,0 +1,22 @@
+using Impostor.Api.Innersloth;
+
+namespace Impostor.Api.Events.Announcements
+{
+    public interface IAnnouncementRequestEvent : IEvent
+    {
+        public interface IResponse
```

Comment:
Don't like Interface inside interface

Reply:
`IAnnouncementRequestEvent.IResponse` is way better than`IAnnouncementRequestResponse`